cmd/ansible.go

```
@@ -148,3 +173,70 @@ var playCmd = &cobra.Command{
 		return nil
 	},
 }
+
+//usage: jarvis ansible list
+//returns:
+// --group(bool) >> list groups name
+// --hosts(string) >> hosts by group
+var inventoryCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Query inventory",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		//checking exclusivity
+		if ListGroup && HostGroupName != "" {
+			return fmt.Errorf("--group and --hosts are mutually exclusive")
+		}
+
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		envsPath := viper.GetString("environments.path")
+
+		allInventories, err := environment.GetFullPathInventoriesFromEnvironments(envsPath, *environments)
+		if err != nil {
+			return err
+		}
+
+		var invReaders []io.Reader
+		for _, invs := range allInventories {
+			for _, inv := range invs {
+				f, err := os.Open(inv)
```

need to check if "inv" is a readable file (and not a directory) b/c it fails silently with directory

---

You are absolutely right :) ... adding this